--- const.go	2022-05-12 01:22:31.000000000 -0700
+++ const_fixed.go	2022-05-12 01:22:24.000000000 -0700
@@ -29,7 +29,7 @@
 	// AttachmentUnused as defined in vulkan/vulkan_core.h:124
 	AttachmentUnused = (^uint32(0))
 	// False as defined in vulkan/vulkan_core.h:125
-	False = uint32(0)
+	False = 0
 	// LodClampNone as defined in vulkan/vulkan_core.h:126
 	LodClampNone = 1000.0
 	// QueueFamilyIgnored as defined in vulkan/vulkan_core.h:127
@@ -41,7 +41,7 @@
 	// SubpassExternal as defined in vulkan/vulkan_core.h:130
 	SubpassExternal = (^uint32(0))
 	// True as defined in vulkan/vulkan_core.h:131
-	True = uint32(1)
+	True = 1
 	// WholeSize as defined in vulkan/vulkan_core.h:132
 	WholeSize = (^uint64(0))
 	// MaxMemoryTypes as defined in vulkan/vulkan_core.h:133
@@ -2221,7 +2221,7 @@
 
 // PipelineCacheHeaderVersion enumeration from https://www.khronos.org/registry/vulkan/specs/1.0/man/html/VkPipelineCacheHeaderVersion.html
 const (
-	PipelineCacheHeaderVersionOne     PipelineCacheHeaderVersion = 1
+	PipelineCacheHeaderVersion1       PipelineCacheHeaderVersion = 1
 	PipelineCacheHeaderVersionMaxEnum PipelineCacheHeaderVersion = 2147483647
 )
 
